src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -65,7 +65,8 @@ struct grpc_tls_key_materials_config
     grpc_core::UniquePtr<char> pem_root_ptr(gpr_strdup(pem_root_certs));
     pem_root_certs_ = std::move(pem_root_ptr);
   }
-  void add_pem_key_cert_pair(grpc_core::PemKeyCertPair pem_key_cert_pair) {
+  void add_pem_key_cert_pair(
+      const grpc_core::PemKeyCertPair& pem_key_cert_pair) {
```

Comment:
Instead of making this a const ref, how about using `std::move()` in the body?  That way, we avoid a copy in cases where people pass an rvalue.

Reply:
That'd work, too.